src/proc/TesseractHandler.ts

```diff
@@ -1,8 +1,102 @@
-import { createWorker } from 'tesseract.js';
+import { createWorker, createScheduler, Worker } from 'tesseract.js';
 import { CaptureArea } from './CaptureArea';
 
 export class TesseractHandler {
-    private stream?: MediaStream;
-    private worker: Worker[] = [];
-    private enabledCaptureAreas: CaptureArea[] = [];
+    private static instance: TesseractHandler;
+    private worker: Worker[];
+    private enabledCaptureAreas: CaptureArea[];
+    private running: boolean;
+
+    /**
+     * Create a private constructor to prevent multiple instances
+     */
+    private constructor() {
+        this.worker = [];
+        this.enabledCaptureAreas = [];
+        this.running = false;
+    }
+
+    /**
+     * This function gets the singelton instance of TesseractHandler.ts
+     *
+     * @return TesseractHandler
+     */
+    public static getInstance(): TesseractHandler {
+        if (!this.instance) {
+            this.instance = new this();
+        }
+
+        return this.instance;
+    }
+
+    public isRunning(): boolean {
```

Comment:
Proper documentation for these functions before merge, and before this PR isn't a draft anymore